src/lib.rs

```diff
@@ -371,6 +372,11 @@ impl Lzxd {
             Ok(view)
         }
     }
+
+    pub fn reset(&mut self) {
```

Comment:
If we want to release new public API, we should document it, even if the method seems straightforward.

Reply:
Done